tests/test_fields.py

```diff
@@ -142,16 +142,16 @@ def test_remove_with_empty_string(self):
         self.assertEqual(serializer.validated_data['created'], uploaded_image.created)
         self.assertIsNone(serializer.validated_data['file'])
 
-    def test_fallback_to_pil_if_not_detected_by_imghdr(self):
+    def test_fallback_to_pil_if_not_detected_by_filetype(self):
         """
-        Passing a sample image which goes undetected by imghdr should
+        Passing a sample image which goes undetected by filetype should
         still be detected by PIL.
         """
         now = datetime.datetime.now()
-        file = UNDETECTABLE_BY_IMGHDR_SAMPLE
+        file = UNDETECTABLE_BY_FILETYPE_SAMPLE
 
-        # check image is undetectable by imghdr
-        self.assertIsNone(imghdr.what('test.jpeg', base64.b64decode(file)))
+        # check image is undetectable by filetype
+        self.assertIsNone(filetype.guess_extension(base64.b64decode(file)))
```

Comment:
This is actually no longer necessary. `self.assertIsNone(filetype.guess_extension(base64.b64decode(file)))` fails by a mistake. It should be formatted like [here](https://github.com/Hipo/drf-extra-fields/blob/b4dfb48b25008954b6ebb18593638ee2becbf9ae/drf_extra_fields/fields.py#L68) before decoding it. I tested it and `filetype` returns `jpg` correctly. So i'm removing this test.